app/src/pages/CardSubmission/SubmissionForm.tsx

```diff
@@ -77,6 +88,8 @@ function SubmissionForm(props: { onSubmit: () => void }) {
                     player_id: currentPlayer.id,
                     game_id: currentGame.id,
                     word: word,
+                    is_allowed:
+                      currentPlayer.role === PlayerRole.Host ? true : null,
```

Comment:
when submitting, if you are the host auto-mark your cards as 👍  -- so they don't show up in review.

Reply:
Makes sense, although maybe not necessary with the conditional host logic here:

https://github.com/avimoondra/fishbowl/blob/5b6497d5fb9f3234e4336d31ba72a5ef56c34aed/app/src/pages/CardSubmission/WaitingForSubmissions.tsx#L53-L56

Should we just leave this `null`?

Reply:
true, I think that's a bit cleaner than marking is_allowed in a sort of synthetic way

Reply:
Addressed in c80b90b3b682ee905b2f76dcb15f2c8b03d8099a.

I left `SubmitCards` as-is:

https://github.com/avimoondra/fishbowl/blob/76da83f5b121e2a4801ae40daf40146ab6808335/app/src/pages/CardSubmission/operations.graphql#L1-L11

but perhaps we don't need to return `is_allowed` anymore?

Reply:
Yup we don't addressed in: https://github.com/avimoondra/fishbowl/pull/98/commits/aeb6e0aa6b30b4c3d97a0da72fbe9a53a9d2d1e0